extra-libs/dpdk/autobuild/prepare

```diff
@@ -0,0 +1 @@
+export CFLAGS="$CFLAGS $(pkg-config cuda --cflags-only-I)"
```

Comment:
abinfo here.

Reply:
Probably not going to be very useful. Will add a comment instead.